app/components/support_interface/reasons_for_rejection_dashboard_component.html.erb

```diff
@@ -0,0 +1,299 @@
+<h3 class="govuk-heading-m govuk-!-font-size-27 govuk-!-margin-top-8">Course full</h3>
+<div id="course_full_section" class="govuk-grid-row govuk-!-margin-bottom-4">
+  <div class="govuk-grid-column-one-half">
+    <div id="headline-stat" class="govuk-!-margin-bottom-4">
+      <%= render SupportInterface::TileComponent.new(
+        count: total_rejection_count("course_full_y_n"),
+        label: 'Total',
+        colour: :blue,
+        size: :reduced,
+      ) %>
+    </div>
+    <div class="govuk-grid-row govuk-!-margin-bottom-4">
+      <div class="govuk-grid-column-one-half">
+        <%= render SupportInterface::TileComponent.new(
+          count: current_month_rejection_count("course_full_y_n"),
+          label: 'this month',
+          size: :reduced,
+        ) %>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <%= render SupportInterface::TileComponent.new(
+          count: previous_rejection_count("course_full_y_n"),
+          label: 'before this month',
+          size: :reduced,
```

Comment:
I hadn't envisioned that we would only have total & last month, but I guess this can't hurt for people who struggle with 'quick maths'.

Reply:
Yeah, this is due to us following the layout of the Feature metrics dashboard and also having the value available so we just thought it wouldn't hurt to display it. We can remove it if it feels a bit too incongruous.

Reply:
I’d be inclined to go with the following layout:

```
[Reason for rejection name]

67%                                 12                16
of all rejections this cycle        last month        this month
```

And have each row stretch the full width of the page (1/2, 1/4, 1/4).

My guess is that having the main number be a percentage of total rejections might be more useful than raw numbers, but I may be wrong.

Reply:
Good idea. The only thing is that it would have to be `% of all structured rejections this cycle` 👍

Reply:
That’s fine, as long as we have a row which captures the remaining unstructured rejections, so the total equals 100%.

Reply:
There is some prior art on percentage for dashboards you may want to refactor/extract
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/support_interface/ucas_matches_statistics_component.rb#L73-L85

Reply:
Thanks @AgaDufrat!